src/data/queries/contactInfo.ts

```diff
@@ -0,0 +1,57 @@
+import { QueryParams, QueryFormatter } from 'next-drupal-query'
+import { ParagraphContactInformation } from '@/types/drupal/paragraph'
+import { ContactInfo } from '@/types/formatted/contactInfo'
```

Comment:
Could make this as FormattedContactInfo for consistency

Reply:
looking at this, the `Type as FormattedType` only comes into play in the templates, the queries all seem to import the type name as-is

Reply:
Yeah, this is a result of naming conflict in the template.